indico/modules/events/tracks/templates/display.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'layout/conference_page_base.html' %}
+
+{% block title %}
+    {{- page_title -}}
```

Comment:
A matter of consistency since there is no block/endblock tags in more common cases like these:

```jinja
<div>
    {{- foo -}}
</div>
```